test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -175,7 +175,6 @@ protected DocumentStore GetDocumentStore(Options options = null, [CallerMemberNa
                         }
 
                         Assert.True(result.RaftCommandIndex > 0); //sanity check             
-                        store.Urls = result.NodesAddedTo.SelectMany(UseFiddler).ToArray();
```

Comment:
Shouldn't we forbid changing the URLs after store is initialized?

Reply:
Yes, I can't think of any reason why we would want to allow that.